### ISSUE TITLE
Loss function in train.py

### ISSUE THREAD

**Issue by SwaggyZhang** [CPF-NLPR/IncrementalED#2]
Thank you for the great job!
During studying the source code, I found something difficult for me to understand. 
### First 
The loss function in 'train.py' is nn.CrossEntropyLoss(). 
```python
logits, _ = self.model(train_X, train_mask)
p = logits[:, :, :self.seen_cls+1]
loss = criterion(p.view(-1, self.seen_cls+1), train_Y.view(-1))
```
The variable 'p' is the first self.seen_cls+1 colums of each matrix in logits tensor, but we usually utilize the index of max value in logits as the prediction result. Could you please tell me why do you choose the self.seen_cls+1 rows as input? And the shape of input is [batch_size * max_length, self.seen_cls+1], which is different with the shape of target 'train_Y.view(-1)'.
### Second
```python
outputs = self.bert(data_x, attention_mask = mask_x)
bert_enc = outputs[0]
logits = self.fc(bert_enc)
return logits, outputs[0]
```
The shape of the variable 'bert_enc' is [batch_size, max_length, hidden_size], and the shape of 'logits' is [batch_size, max_length, y_num]. Do you want to get the pooled result of bert encoder and the corresponding predictions?
